moveit_core/collision_detection/src/collision_tools.cpp

```diff
@@ -39,7 +39,7 @@
 #include <rclcpp/time.hpp>
 #include <tf2_eigen/tf2_eigen.hpp>
 
-namespace collision_detection
+namespace collision_detection // This namespace is perfectly pure, nothing to be modified actually, perfect.
```

Comment:
Can you move this comment to the header file, "collision_tools.h"? I suggest this, I think it reads more easily:

```
namespace collision_detection
{
// Perfectly "pure" helper functions
```